packages/pyroscope-flamegraph/src/FlameGraph/FlameGraphComponent/index.spec.tsx

```diff
@@ -22,6 +22,7 @@ describe('FlamegraphComponent', () => {
 
     render(
       <FlamegraphComponent
+        showCredit
```

Comment:
if it's optional, why did you have to add it here?

Reply:
it's optional for the parent component, which is [FlameGraphRenderer](https://github.com/pyroscope-io/pyroscope/pull/1219/files#diff-6a469ade99d60ee9e8195ee6e6e0a707f7ab092dfc7d1abd3242260853e07d8f), with default `true`. passing `showCredit` here is the emulation of parent default prop

Reply:
oh that makes sense!